src/Config/Config.js

```diff
@@ -132,4 +132,13 @@ export default {
     readOnly: "readOnly",
     toggleNoView: "toggleNoView"
   },
-};
\ No newline at end of file
+  AnnotationProperties: {
```

Comment:
not sure what are the usage of these?

Reply:
So they could use them through JSON.parse()? If it is unrealistic, I could remove them and update README

Reply:
but this is not actually used in code tho?